### ISSUE TITLE
[BUG] [Formatter] Custom template tags beginning with 'set' breaks the tag

### ISSUE THREAD

**Issue by mfisco** [Riverside-Healthcare/djLint#634]
- [X] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [X] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [X] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS Ventura 13.3.1 (a)
 - Python Version: 3.10.6
 - djLint Version: >= 1.25.1
 - template language: Django

## Issue

A custom [simple_tag](https://docs.djangoproject.com/en/4.2/howto/custom-template-tags/#simple-tags) that's prefixed with _set_ (e.g., _setting_, _setdate_, etc.) splits the tag's name.
  > If you downgrade back to version 1.25.0, the formatter works as expected. Possibly related to #557 included in the 1.25.1 release.

## How To Reproduce

1. Create a template that's _already_ formatted:
    ```html
    <title>{% set_title "My Title" %}</title>
    ```
2. Running `djlint --reformat` should have no effect, but gives this output:
    ```html
    <title>{% set _title "My Title" %}</title>
    ```
